pom.xml

```diff
@@ -143,6 +147,12 @@ THE SOFTWARE.
       <artifactId>powermock-api-mockito</artifactId>
       <version>1.5</version>
       <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>org.mockito</groupId>
+          <artifactId>mockito-all</artifactId>
```

Comment:
Ideally this dependency should be removed at all from the plugin. Mockito Core should be enough
https://github.com/mockito/mockito/wiki/Declaring-mockito-dependency